src/utils/mail/SendingUtils.ts

```diff
@@ -0,0 +1,25 @@
+import {google} from 'googleapis'
+
+export default class SendingUtils {
```

Comment:
Looks fine but could you elaborate on why you opted to write a class here? It doesn't hold any internal state or need to be instantiated as an object.